sonoff rfbridge direct hack - first commit

brahmafear opened the pull request:
Adds environment directive to platform.io for Sonoff RF Bridge modified with "direct hack" allowing bypass of efm8bb1 microcontroller.  ( https://github.com/xoseperez/espurna/wiki/Hardware-Itead-Sonoff-RF-Bridge---Direct-Hack ).

1technophile commented:
Thanks for the PR, could you isolate the change to the added environment, please.
It will make the diff more comprehensive and avoid regressions.

brahmafear commented:
Moved new environment to end of file to prevent other changes.  Removed "uncomment" listing for this environment from top section.  Sorry, I'm not very good with git.

1technophile commented:
No problem, you need also to add
`;default_envs = sonoff-rfbridge-direct`
at the end of the environment list (beginning of the file)

1technophile commented:
Perfect, thanks

1technophile commented on platformio.ini:
```diff
@@ -871,3 +872,22 @@ build_flags =
+lib_deps = 
+	${com-esp.lib_deps}
+	${libraries.esp8266_mdns}
+	puuu/ESPiLight@^0.17.0
```
Could you call
`  ${libraries.esppilight}`
instead please

1technophile commented:
Just one comment :-)

1technophile commented on platformio.ini:
```diff
@@ -871,3 +872,22 @@ build_flags =
+lib_deps = 
+	${com-esp.lib_deps}
+	${libraries.esppilight}
+	puuu/ESPiLight@^0.17.0
```
You are right mdns is not necessary here.
You should also remove : 
```
puuu/ESPiLight@^0.17.0
```
Since it is called by 
```
${libraries.esppilight}
```

1technophile commented:
Thanks !
